currency_rate_inverted/README.rst

```diff
@@ -23,7 +23,7 @@ Currency Rate Inverted
     :target: https://runbot.odoo-community.org/runbot/259/13.0
     :alt: Try me on Runbot
 
-|badge1| |badge2| |badge3| |badge4| |badge5| 
+|badge1| |badge2| |badge3| |badge4| |badge5|
```

Comment:
Just revert the file entirely

Reply:
no problem about this, OCA-git-bot will amend it anyways

Reply:
That is also true. This is a non-blocking request :) Question is only about purity, so no strong feelings about this

Reply:
it was done by precommit. I can revert if it is needed